EIPS/eip-5267.md

```diff
@@ -4,7 +4,7 @@ title: Retrieval of EIP-712 domain
 description: A way to describe and retrieve an EIP-712 domain to securely integrate EIP-712 signatures.
 author: Francisco Giordano (@frangio)
 discussions-to: https://ethereum-magicians.org/t/eip-5267-retrieval-of-eip-712-domain/9951
-status: Draft
+status: Review
 type: Standards Track
 category: ERC
 created: 2022-07-14
```

Comment:
Typo on line 96: json5 should be json

Otherwise lgtm

Reply:
It's not a typo: https://json5.org/

Reply:
Okay then, fair enough.